spec/nylas/free_busy_collection_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Nylas::FreeBusyCollection do
+  describe "#each" do
+    it "returns collection of `FreeBusy` objects" do
+      start_time = 1_609_439_400
+      end_time = 1_640_975_400
+      emails = ["test@example.com", "anothertest@example.com"]
+      api = instance_double("Nylas::API")
+      free_busy_response = [
+        {
+          object: "free_busy",
+          email: "test@example.com",
+          time_slots: [
+            {
+              object: "time_slot",
+              status: "busy",
+              start_time: 1_609_439_400,
+              end_time: 1_640_975_400
+            }
+          ]
+        },
+        {
+          object: "free_busy",
+          email: "anothertest@example.com",
+          time_slots: []
+        }
+      ]
+      allow(api).to receive(:execute).and_return(free_busy_response)
+      result = described_class.new(
+        api: api,
+        emails: emails,
+        start_time: start_time.to_i,
+        end_time: end_time.to_i
+      ).to_a
+
+      expect(result.size).to eq(2)
+      first_result = result.first
```

Comment:
Is there a reason not to iterate over all of the objects in this array and test them? I'm assuming performance?

Reply:
We just have one object so we don't really need to loop over things here.